presto-main/src/main/java/com/facebook/presto/connector/system/jdbc/ProcedureColumnJdbcTable.java

```diff
@@ -32,7 +32,7 @@
 
     public static final ConnectorTableMetadata METADATA = tableMetadataBuilder(NAME)
             .column("procedure_cat", createUnboundedVarcharType())
-            .column("procedure_schem", createUnboundedVarcharType())
+            .column("procedure_schema", createUnboundedVarcharType())
```

Comment:
Can you pull this change out?  it needs a release note since it's a column name in a table we expose.  I also want to double check whether there's anything else we need to do to make this change.